### DISCUSSION TITLE
Copilot toggle on/off

### DISCUSSION THREAD

**Discussion by rileyblackwell** [community/community#49109] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Great product, but adding a keyboard shortcut to toggle copilot on/off would be a very nice feature. There are times when you want to temporarily ignore Copilot's suggestions because they aren't accurate/useful. It would be less distracting to simply toggle copilot off and then when you reach a line of code where copilot is useful again toggle back on.

**Comment by tyler-holm** [comment]
There is no default key binding for this but you can actually set one up yourself. Either find it in the GUI settings or set in manually with the key github.copilot.toggleCopilot

EDIT: this should help
https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio-code

**Comment by rileyblackwell** [comment]
The problem with the current toggle on/off is that a window pops up asking if you want to disable Copilot globally or for a particular language. Of course, this causes you to have to use the mouse and efficiency is lost. A toggle on/off keyboard shortcut that immediately disables copilot in the current workspace would be ideal.

**Comment by amuhareb** [comment]
it is one of the worst design decisions i have come across in many many months. I am assuming that this is, of course, intentional. That the goal is to obstruct user workflow so that in a few months they can introduce a new feature that makes intellisense and other such extensions obsolete. As a budding developer, I'm just beginning to discover the function tooltip windows and they are proving essential. but so are the training wheels of copilot. 

if i'm wrong, i'd love for a copilot team member to tell me why this is impossible.

**Comment by amuhareb** [comment]
@github, seriously, what the fuck?
